content/browser/frame_host/render_frame_host_impl.cc

```diff
@@ -870,6 +870,11 @@ void RenderFrameHostImpl::AddMessageToConsole(ConsoleMessageLevel level,
   Send(new FrameMsg_AddMessageToConsole(routing_id_, level, message));
 }
 
+void RenderFrameHostImpl::InsertUserStylesheet(const std::string& stylesheet) {
```

Comment:
Would be good to have a comment (either here or in the header file), pointing to the issue for future reference why this was added to the upstream code

Reply:
ACK